src/platform/Linux/xdg/iconlookup.cpp

```diff
@@ -35,7 +35,12 @@ XDG::IconLookup::IconLookup()
         if (QFile::exists(path = QDir(basedir).filePath("icons")))
             iconDirs_.append(path);
 
+#if defined Q_OS_FREEBSD
+    path = "/usr/local/share/pixmaps";
```

Comment:
the spec explicitly tells to look at /usr/share/pixmaps
https://specifications.freedesktop.org/icon-theme-spec/latest/ar01s03.html

Reply:
Which does not exist on FreeBSD because of LOCALBASE fsh..

Reply:
mhm okay, although not standardized i see no realy problem adding it.